src/tinypeel/tinypeel.py

```diff
@@ -357,22 +385,28 @@ def getArgs():
     )  # help='An optional external penetrance file. This will overwrite the default penetrance values.')
     InputOutput.add_arguments_from_dictionary(
         peeling_parser,
-        InputOutput.get_probability_options(),
-        options=["error", "seqerror"],
+        get_probability_options(),
+        options=["geno_error_prob", "seq_error_prob"],
     )
 
     peeling_control_parser = parser.add_argument_group("Peeling control arguments")
     peeling_control_parser.add_argument(
-        "-esterrors",
+        "-est_geno_errors_prob",
         action="store_true",
         required=False,
         help="Flag to re-estimate the genotyping error rates after each peeling cycle.",
     )
     peeling_control_parser.add_argument(
-        "-estmaf",
+        "-est_seq_errors_prob",
+        action="store_true",
+        required=False,
+        help="Flag to re-estimate the sequencing error rates after each peeling cycle.",
+    )
+    peeling_control_parser.add_argument(
+        "-est_alt_allele_prob",
         action="store_true",
         required=False,
-        help="Flag to re-estimate the minor allele frequency after each peeling cycle.",
+        help="Flag to re-estimate the alternative allele probability after each peeling cycle.",
```

Comment:
@AprilYUZhang change to `probabilities` since we do it for multiple loci. Check also elsewhere.